apps/bare-expo/package.json

```diff
@@ -86,4 +86,4 @@
     "type": "all",
     "jsSrcsDir": "./modules"
   }
-}
+}
```

Comment:
I wonder why the trailing line gets removed 🤔 Maybe you have some custom formatting options in your IDE?

Reply:
Correct - needed to add this to my VS Code settings file.

```
"json.format.keepLines": true,
```